Kinvey-Xamarin/Client/Client.cs

```diff
@@ -194,7 +194,7 @@ public Builder(string appKey, string appSecret)
 				Credential currentCredential = this.Store.GetStoredCredential(this.ssoGroupKey);
 				if (currentCredential != null)
 				{
-					await User.LoginAsync(currentCredential, c);
+					User.LoginAsync(currentCredential, c);
```

Comment:
Effectively, it is.  The `LoginAsync(credential)` call never makes a network request, so it never calls an `await`-able method.  At some point, as a refactoring exercise, I would like to perhaps make this method call a bit more explicit and readable, but any of those changes will be under the hood, and will not change the `Client.Builder Build()` API.